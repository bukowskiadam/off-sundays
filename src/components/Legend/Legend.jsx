import React from 'react';

import StatusImage from '../StatusImage';

import style from './Legend.module.css';

const Legend = () => (
    <div className={style.container}>
        <div className={style.entry}>
            <StatusImage isClosed={true} />
            <span>handel ograniczony</span>
        </div>
        <div className={style.entry}>
            <StatusImage isClosed={false} />
            <span>handel dozwolony</span>
        </div>
    </div>
);

export default Legend;
