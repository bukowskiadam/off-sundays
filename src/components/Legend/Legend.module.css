.container {
    margin-bottom: 2rem;
}

.entry {
    font-size: 60%;
    line-height: 2em;
    margin-bottom: 0.2em;
    display: inline-block;
    width: 140px;
}

.entry img {
    height: 2em;
    vertical-align: middle;
    margin: 0;
    padding-right: 0.5em;
}
