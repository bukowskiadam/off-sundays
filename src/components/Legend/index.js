import React from 'react';

import StatusImage from '../StatusImage';

import style from './index.module.css';

export default () => (
    <div className={style.container}>
        <div className={style.entry}>
            <StatusImage isClosed={true} />
            <span>handel ograniczony</span>
        </div>
        <div className={style.entry}>
            <StatusImage isClosed={false} />
            <span>handel dozwolony</span>
        </div>
    </div>
);
