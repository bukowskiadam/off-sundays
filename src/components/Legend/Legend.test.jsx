import React from 'react';
import { shallow } from 'enzyme';

import Legend from './Legend';

describe('Legend', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<Legend />);

        expect(wrapper).toMatchSnapshot();
    });
});
