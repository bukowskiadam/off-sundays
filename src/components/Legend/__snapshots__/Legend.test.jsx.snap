// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Legend renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="entry"
  >
    <StatusImage
      isClosed={true}
    />
    <span>
      handel ograniczony
    </span>
  </div>
  <div
    className="entry"
  >
    <StatusImage
      isClosed={false}
    />
    <span>
      handel dozwolony
    </span>
  </div>
</div>
`;
